tests/unit/test_vcs.py

```diff
@@ -177,3 +177,21 @@ def test_bazaar_simple_urls():
 def test_get_git_version():
     git_version = Git().get_git_version()
     assert git_version >= parse_version('1.0.0')
+
+
+@pytest.mark.parametrize('url, expected', [
+    # Test a basic case.
+    ('svn+https://svn.myproject.org/MyProject#egg=MyProject',
+     ('svn+https://svn.myproject.org/MyProject#egg=MyProject', [])),
+    # Test with username and pass.
+    ('svn+https://user:pass@svn.myproject.org/MyProject#egg=MyProject',
+     ('svn+https://svn.myproject.org/MyProject#egg=MyProject',
+      ['--username', 'user', '--password', 'pass'])),
+])
+def test_subversion__get_url_rev_args(url, expected):
```

Comment:
Should we also have non-svn tests for this change?

Reply:
Thanks, and sure! For non-SVN the implementation is "trivial," but it certainly can't hurt to add. I'll do that now.